content/index.rst

```diff
@@ -4,173 +4,126 @@ grid_template: true
 image:
   - src: 'pages/faces-hero.jpg'
 headline:
-  - title: 'You have a brilliant idea.'
-    tagline: "We'll make it fly."
-    type: 'Oddly Human Design & Development'
+  - title: 'We help you meet your goals'
+    type: "We don't just write the code…"
 summary: |
-  **Technology is only useful
-  when it solves real problems for real people.**
-  As your partner,
-  we'll help find those opportunities –
-  providing `web application design and development`_,
-  with a human focus from start to finish.
+  **We provide the full range of**
```

Comment:
That's a lot of "We..."

![screenshot 2017-10-30 15 21 07](https://user-images.githubusercontent.com/552316/32191027-05f93030-bd86-11e7-8c18-95e08adda2a1.jpg)

Reply:
Maybe "OddBird provides…" for the main text?